### DISCUSSION TITLE
Make a Language selector in the Website

### DISCUSSION THREAD

**Discussion by MAVRICK-1** [InternetHealthReport/gsoc#32] [Ideas]
Website language has a direct impact on the [user](https://abmatic.ai/blog/benefits-of-user-generated-content-for-growth) experience, which is why it's so important to get it right. If a user lands on your website and can't understand the language, they're likely to quickly leave and move on to another site. On the other hand, if the language is easy to understand and well-written, it can help create a positive user experience that keeps visitors on your site and engaged with your content. The language you use can also impact the overall design and feel of your website, affecting the way users perceive your brand and your products.

**Comment by romain-fontugne** [comment]
Thanks, indeed it would be nice to have the website in other languages but, first, this is not a priority I think we have a lot of other things to work on, and the main difficulty is not the selector but translating all texts to a different language. I think that's where we need most help
